src/applications/static-pages/helpers.js

```diff
@@ -0,0 +1,13 @@
+export function qSA(rootNode, selector) {
+  return Array.from(rootNode.querySelectorAll(selector));
+}
+
+export function qS(rootNode, selector) {
```

Comment:
did you make this just to be consistent with `qSA`?

Reply:
This, like qSA, seems to be commonly used by these widget creators and their tests, so I've added it here so it can be shared.

Reply:
I was just wondering what the benefit of this wrapper around the native `querySelector` was, since you don't need to transform the data like you do with the `qSA` helper. The biggest benefit seems to be that it's more consistent to use `qS` since we're going to be using `qSA` as well.